docs/source/connectors.rst

```diff
@@ -0,0 +1,113 @@
+Connectors
+==========
+Connect your preprocessing pipeline with your favorite data storage platforms, and batch process all your documents using the provided CLI to store structured outputs locally on your filesystem. 
+
+To use any connector you'll need to first install ``local-inference``. You can find more information about this in the `installation guide <https://unstructured-io.github.io/unstructured/installing.html>`_.
```

Comment:
Small adjustment on this, you only need the `local-inference` extra if you're processing PDFs or images. However, you _do_ need the extra associated with the connector you're trying to us. So you'd need `pip install "unstructured[s3]"` or `pip install "unstructured[gcs,local-inference]"` if you have PDFs/images and want to use the GCS connector.